payloads/library/execution/keyboard_hangover/payload.txt

```diff
@@ -0,0 +1,53 @@
+# Title: keyboard_hangover
+# Description: keyboard_hangover is a powerful payload which randomly remaps targets keyboard.
+# AUTHOR: drapl0n
+# Version: 1.0
+# Category: Execution.
+# Target: GNU/Linux.
+# Attackmodes: HID.
+
+LED SETUP
+ATTACKMODE STORAGE
```

Comment:
Your payload is doing keystroke injection, I would change this to `ATTACKMODE HID STORAGE` so the bash bunny can identify as both.